### PR TITLE
Code owners citadel style

### PR DESCRIPTION
**Pull request by Poojawa**
Because it's neat and should be a thing on our code to prevent ghost nerfs.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,67 +4,15 @@
...
-/_maps/ @ShizCalev
-/sound/ @ShizCalev
-
-# Multiple Owners
-
-/code/controllers/subsystem/air.dm @duncathan @MrStonedOne
-/icons/ @ShizCalev @WJohn
+#deathride58
+/modular_citadel/ @deathride58
+/code/citadel/ @deathride58
```
Comment (deathride58) [inline .github/CODEOWNERS]: You can probably change this to just /code/ so that I can scream at anyone who thinks of making non-modular changes

Comment (Poojawa) [reply .github/CODEOWNERS]: which will end up being every single mirror'd PR.

Comment (deathride58) [reply .github/CODEOWNERS]: Yes

**Review by deathride58 — DISMISSED** [review]
Could probably add Jay to __donator.dm in the loadout gear stuff and custom_items.dm so that it'll automatically request a review from Jay for any donor fluff that gets added

**Review by deathride58 — APPROVED** [review]

**Pull request merged**